### PR TITLE
Add template styles

### PR DESCRIPTION
**Pull request by vegeorgi**
- Add css file to .gitignore
 - Add page background icons files
 - Add nativescript-dev-sass and node-sass to package.json

### CODE REVIEW

File: platform.ios.scss [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+// Start custom common variables
+@import 'sass/app-variables';
+@import 'sass/app-classes';
+// End custom common variables
+
+@import '~nativescript-theme-core/scss/platforms/index.ios';
+
+// Place any CSS rules you want to apply only on iOS here
+
+.homePage {
```
Comment (MIYankov) [inline platform.ios.scss]: These are component specific and should go into each component's specific scss file.

File: platform.android.scss [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+// Start custom common variables
+@import 'sass/app-variables';
+@import 'sass/app-classes';
+// End custom common variables
+
+@import '~nativescript-theme-core/scss/platforms/index.android';
+
+// Place any CSS rules you want to apply only on Android here
+.homePage {
```
Comment (MIYankov) [inline platform.android.scss]: These are component specific and should go into each component's specific scss file.

File: sass/_app-classes.scss [hunk 3]
```diff
@@ -0,0 +1,9 @@
+.fa {
```
Comment (MIYankov) [inline sass/_app-classes.scss]: These common classes should be moved to the app.scss, imho. That's the idea of the file.

File: shared/my-drawer/MyDrawer.xml [hunk 4]
```diff
@@ -2,10 +2,22 @@
...
+<StackLayout class="sidedrawer sidedrawer-left" loaded="onLoaded">
+    <StackLayout class="sidedrawer-header">
+        <Label class="sidedrawer-header-image fa" text="&#xf2bd;"></Label>
+        <Label class="sidedrawer-header-brand" text="User Name"></Label>
+        <Label class="footnote" text="username@mail.com"></Label>
+    </StackLayout>
+
+    <StackLayout class="sidedrawer-content">
+        <ListView id="listview" separatorColor="transparent" items="{{ navigationItems }}" itemTap="onNavigationItemTap">
         <ListView.itemTemplate>
```
Comment (vegeorgi) [inline shared/my-drawer/MyDrawer.xml]: 👍

File: platform.ios.scss (hunk 1 above)
Comment (vegeorgi) [inline platform.ios.scss]: 👍

File: platform.android.scss (hunk 2 above)
Comment (vegeorgi) [inline platform.android.scss]: 👍

File: sass/_app-classes.scss (hunk 3 above)
Comment (vegeorgi) [inline sass/_app-classes.scss]: 👍

**Pull request merged**